### PR TITLE
teamlist.md: Add teamlist as specified in cEP-0003

### PR DESCRIPTION
**Pull request by sims1253**

### CODE REVIEW

File: teamlist.md
Comment (sils) [inline teamlist.md]: I'd remove this line. Or add a normal text like, for more information about the teams, check out the cep

File: teamlist.md
Comment (sils) [inline teamlist.md]: why the stuff in quotes?

File: teamlist.md
Comment (sils) [inline teamlist.md]: still not sure, we should IMO also reevaluate if we really want this migration

**Review by sils — CHANGES_REQUESTED** [review]

File: teamlist.md
Comment (adtac) [inline teamlist.md]: `hosted`? Never heard this word and Google doesn't return any definition

File: teamlist.md
Comment (adtac) [reply teamlist.md]: Maybe form the team and let the leader make a proposal (in cEP) and we can review it? Would be worth it to see the pros and cons documented and discussed IMO

Comment (adtac) [reply teamlist.md]: And if it's not worth it, we can always dissolve the team.

File: teamlist.md
Comment (adtac) [inline teamlist.md]: I'm not sure if we really need a team for testing - it should go in hand with the library. We don't merge PRs that decrease coverage anyway, so whoever writes the module will have to write the test too. So this will basically be a union of all programming related teams :P

**Review by adtac — COMMENTED** [review]
So I'm just wondering - which team is "responsible" for coala-quickstart, coala-html and the others? Core?

File: teamlist.md
Comment (sims1253) [reply teamlist.md]: so far we don't have an application for this anyway so we could remove it and if someone is interested he/she has to apply again. Didn't @jayvdb kinda work on this?

File: teamlist.md
Comment (sims1253) [reply teamlist.md]: It's one of the teams from the first draft made with @sils 
I like the idea of a "QA" department but guess you are right and we could remove this.

**Comment by sims1253** [comment]
so far no team is responsible for the things @adtac and thus by default the core team kinda is.

**Comment by sims1253** [comment]
@adtac ATM the cEP states that a team either has to be proposed in this repo or is done so implicitly via an application for teamlead. As there is an application by adrian, there now is a cib team.

File: teamlist.md
Comment (sils) [inline teamlist.md]: didn't we want to remove this team?

**Review by sils — DISMISSED** [review]

File: teamlist.md
Comment (sims1253) [reply teamlist.md]: I'm about to remove all teams without an application just so this gets merged :D 
Will remove.

**Comment by sils** [comment]
sure go ahead

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala.io/ - http://viperdev.io/ - http://gitmate.com/

**Comment by Mixih** [comment]
@sims1253 What is the status of this pull?

**Comment by sims1253** [comment]
@Mixih I am waiting for acks so I can finally merge it :D

**Comment by sils** [comment]
ack https://github.com/coala/teams/pull/5/commits/d92a22752ef507211977ea08263b6729d94639aa https://github.com/coala/teams/pull/5/commits/62083fa24762d5f111155dfa7619cc40a9945b06

**Comment by sils** [comment]
ack d92a227 62083fa

**Comment by sils** [comment]
ack d92a22752ef507211977ea08263b6729d94639aa 62083fa24762d5f111155dfa7619cc40a9945b06

**Comment by sims1253** [comment]
@rultor merge

**Comment by Mixih** [comment]
We need a rultor file for this repo.

**Pull request merged**
